### PR TITLE
[cache_keycloak_token] | cache keycloak token.

### PR DESCRIPTION
**Pull request by MounicaSagi**

### CODE REVIEW

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=6006902&bid=19691706)

File: src/main/java/in/projecteka/consentmanager/ConsentManagerConfiguration.java [hunk 1]
```diff
@@ -303,7 +307,7 @@ public UserServiceClient userServiceClient(
     }
 
     @Bean
-    public OtpServiceClient otpServiceClient(@Qualifier("customBuilder") WebClient.Builder builder,
+    public OtpServiceClient otpServiceClient(@Qualifier("customBuilder") Builder builder,
```
Comment (ganesanarun) [inline src/main/java/in/projecteka/consentmanager/ConsentManagerConfiguration.java:310]: It will not be obvious, what builder it's. Keep that change.

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=6006902&bid=19691706)

File: src/main/java/in/projecteka/consentmanager/ConsentManagerConfiguration.java (hunk 1 above)
Comment (MounicaSagi) [reply src/main/java/in/projecteka/consentmanager/ConsentManagerConfiguration.java]: Done.

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=6006902&bid=19691706)

**Pull request merged**